tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -93,7 +93,7 @@ spec:
               tag="${prefixedTag}"
           else
               defaultTag=$(jq -r '.images.defaultTag // "latest"' "${DATA_FILE}")
-              tag=$(jq -r '.tag // "${defaultTag}"' <<< $component)
+              tag=$(jq -r --arg tag $defaultTag '.tag // $tag' <<< $component)
```

Comment:
For the record, the reason `jq -r '.tag // "${defaultTag}"'` wouldn't work (as suggested in the chat) is that the default value needs to be a string. And a string needs `""` around it in json world. So this would have worked:

`jq -r ".tag // \"${defaultTag}\""`

But I like the version with `--arg` - I didn't know you could define vars like this.

Reply:
Ah, nice. I am jealous of how well you know `jq`

Reply:
I don't think I know it well ;) With the default value, when I was reviewing (I think) your PR where you needed a default and used some `if`s, I just thought "hmm, I wonder if `jq` has some option for a default value" and checked the documentation and voila :)